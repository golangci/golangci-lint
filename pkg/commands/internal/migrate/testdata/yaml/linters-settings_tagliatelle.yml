issues:
  # Only to not generate unrelated elements inside golden.
  exclude-use-default: false
  # Only to not generate unrelated elements inside golden.
  exclude-generated: strict
  # Only to not generate unrelated elements inside golden.
  exclude-dirs-use-default: false

linters-settings:
  tagliatelle:
    case:
      rules:
        json: camel
        yaml: camel
        xml: camel
        toml: camel
        bson: camel
        avro: snake
        mapstructure: kebab
        env: upperSnake
        envconfig: upperSnake
        whatever: snake
      extended-rules:
        json:
          case: camel
          extra-initialisms: true
          initialism-overrides:
            DB: true
            LHS: false
      use-field-name: true
      ignored-fields:
        - Bar
        - Foo
      overrides:
        -
          pkg: foo/bar
          rules:
            json: snake
            xml: pascal
          extended-rules:
          use-field-name: true
          ignored-fields:
            - Bar
            - Foo
          ignore: true
