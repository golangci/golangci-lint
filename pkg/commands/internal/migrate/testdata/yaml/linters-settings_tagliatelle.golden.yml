version: "2"
linters:
  settings:
    tagliatelle:
      case:
        rules:
          avro: snake
          bson: camel
          env: upperSnake
          envconfig: upperSnake
          json: camel
          mapstructure: kebab
          toml: camel
          whatever: snake
          xml: camel
          yaml: camel
        extended-rules:
          json:
            case: camel
            extra-initialisms: true
            initialism-overrides:
              DB: true
              LHS: false
        use-field-name: true
        ignored-fields:
          - Bar
          - Foo
